lib/usd/utils/util.cpp

```diff
@@ -137,4 +186,23 @@ hasSpecs(const UsdPrim& prim)
     return found;
 }
 
+
+void
+printCompositionQuery(const UsdPrim& prim)
```

Comment:
Might be useful to pass in an std::ostream&, instead of forcing use of std::cout.

Reply:
@ppt-adsk please see de4ed7c4b7dda2c6c78ba34b618e29d088bdd0fe